transcription/transcription.go

```diff
@@ -11,13 +11,15 @@ import (
 
 // SendEmail connects to an email server at host:port, switches to TLS,
 // authenticates on TLS connections using the username and password, and sends
-// an email from address from, to address to, with subject line subject with message body.
+// an email from address from, to address to, with subject line subject with
+// message body.
 func SendEmail(username string, password string, host string, port int, to []string, subject string, body string) error {
```

Comment:
So Effective Go says
> Go has no line length limit. Don't worry about overflowing a punched card. If a line feels too long, wrap it and indent with an extra tab.

I read or was told once to try to keep comments below 80 characters though. I think it's our call if we want to break the line up.